ocaml/stdlib/obj.mli

```diff
@@ -42,6 +42,14 @@ external reachable_words : t -> int = "caml_obj_reachable_words"
      @since 4.04
   *)
 
+val uniquely_reachable_words : t array -> int array
+(** For each element of the array, computes the total size (as defined
+    above by [reachable_words]) of all heap blocks accessible from the
+    argument but excluding all blocks accessible from any other arguments.
+
+    @since 4.15
```

Comment:
I don't think there will ever be a 4.15! (Upstream is on 5 now)

Reply:
What should I put in the since field? I thought 4.14 might be misleading since it was not available right from the release. Would it be appropriate to omit it entirely?